### PR TITLE
Update README and testflight instruction files

### PR DESCRIPTION
**Pull request by marionbarker**
~~This PR has one line (about uncommenting the schedule and cron jobs) that depends on PR #38, but can be merged independently.~~ See next conversation about why the one line was removed.

* Added link from README to testflight for GitHub build instructions.
* Modified some instructions based on experience with GitHub build.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # LoopWorkspace
 
+## New GitHub Build Instructions
+
+The new GitHub Build Instructions are at this [link](fastlane/testflight.md)
```
Comment (ps2) [inline README.md:5]: These two options should not be presented as "new" vs "traditional", but rather "for developers, with full build control", vs "for non-developers"

Comment (ps2) [reply README.md:5]: They're both "current" methods; the github based build does not supercede the xcode based method.

File: fastlane/testflight.md [hunk 2]
```diff
@@ -1,53 +1,106 @@
...
+These instructions allow you to build Loop without having access to a Mac.
 
-The setup steps are somewhat involved, but nearly all are one time steps. Subsequent builds are trivial.  Note that TestFlight requires apple id accounts 13 years or older. Your app must be updated once every 90 days, but it's a simple click to make a new build and can be done from anywhere.
+* You can install Loop on phones via TestFlight that are not connected to your computer
+* You can send builds and updates to those you care for
+* You can install Loop on your phone using only the TestFlight app if a phone was lost or the app is accidentally deleted
+* You do not need to worry about specific Xcode/Mac versions
+
+An app built using this method can easily be deployed to newer versions of iOS, as soon as they are available.
```
Comment (ps2) [inline fastlane/testflight.md:10]: This makes it sound like you'd need to re-deploy if you update iOS.

File: README.md (hunk 1 above)
Comment (ps2) [reply README.md:5]: Maybe link to the fastlane/testflight doc with something like. "If you're not a developer, or are looking for a way to automate builds to TestFlight, ..."

Comment (marionbarker) [reply README.md]: Modified README. Looking at testflight next.

File: fastlane/testflight.md (hunk 2 above)
Comment (marionbarker) [reply fastlane/testflight.md]: The last sentence were your original words, but taken out of context. I'm not sure they are needed.
Pushed a commit. Let me know if you want additional changes.

**Pull request merged**